pkg/kubelet/metrics/metrics.go

```diff
@@ -623,7 +623,7 @@ var (
 		&metrics.CounterOpts{
 			Subsystem:      KubeletSubsystem,
 			Name:           StartedHostProcessContainersErrorsTotalKey,
-			Help:           "Cumulative number of errors when starting hostprocess containers. This metric will only be collected on Windows and requires WindowsHostProcessContainers feature gate to be enabled.",
+			Help:           "Cumulative number of errors when starting hostprocess containers.",
```

Comment:
does the `This metric will only be collected on Windows` bit still apply?

Reply:
Good catch! The metric will still only be collected on Windows nodes.
Let me update this.

Reply:
updated